applications/nrf_desktop/README.rst

```diff
@@ -205,7 +205,15 @@ Every of these reports uses predefined report format and provides the given info
 For example, the mouse motion is forwarded as HID mouse report.
 
 An nRF Desktop device supports the selected subset of the HID input reports.
-For example, the nRF Desktop keyboard reference desing (nrf52kbd_nrf52832) supports HID keyboard report, HID consumer control report and HID system control report.
+For example, the nRF Desktop keyboard reference design (nrf52kbd_nrf52832) supports HID keyboard report, HID consumer control report and HID system control report.
+
+The following HID protocols are supported by nRF Desktop for HID input reports:
+
+* Report protocol - Most widely used in HID devices.
+   When establishing connection, HID device defines the device data in a HID descriptor and sends the data to the host.
+* Boot protocol - Only available for mice and keyboards.
+   No HID descriptor is used for this HID protocol.
+   Instead, fixed data packet formats must be used to send data from the HID device to the host.
```

Comment:
are used to exchange data between HID device and host (keep in mind that we also have boot keyboard LEDs output report)